llama_stack/apis/tools/tools.py

```diff
@@ -130,8 +130,10 @@ async def unregister_tool_group(self, tool_group_id: str) -> None:
 class ToolRuntime(Protocol):
     tool_store: ToolStore
 
-    @webmethod(route="/tool-runtime/discover", method="POST")
-    async def discover_tools(self, tool_group: ToolGroupDef) -> List[ToolDef]: ...
+    @webmethod(route="/tool-runtime/list-tools", method="GET")
+    async def list_runtime_tools(
```

Comment:
can you add a TODO here saying this could be named `list_tools()` but there is some error with our OpenAPI spec generator